tests/base/src/test/java/org/wildfly/security/sasl/scram/BasicScramSelfTest.java

```diff
@@ -265,6 +241,17 @@ public void testPlusClientWithBindingWithNonPlusServer() throws Exception {
         }
     }
 
+    private void performAuthenticationTest(String username, String password) throws Exception {
+        final SaslServer saslServer =
+                new SaslServerBuilder(ScramSaslServerFactory.class, SaslMechanismInformation.Names.SCRAM_SHA_1)
+                        .setUserName("username")
```

Comment:
The `username` variable needs to be used instead of the string `"username"`, e.g., `setUserName(username)`.

Reply:
ooh ok got it now :+1:.